app/_locales/en/messages.json

```diff
@@ -1781,7 +1781,7 @@
     "description": "$1 is the number of quotes that the user can select from when opening the list of quotes on the 'view quote' screen"
   },
   "swapNetworkFeeSummary": {
-    "message": "The network fee covers the cost of processing your swap and storing it on the Ethereum network. MetaMask does not profit from this fee."
+    "message": "The network fee covers the cost of processing your swap and storing it on the $1 network. MetaMask does not profit from this fee."
```

Comment:
Note that this will break for any translations that don't include a `$1`. I'd suggest going through each translation to either replace `Ethereum` with `$1`, or if that seems difficult then deleting this string so that we can ask for a new translation.

Reply:
I've talked with @danjm and he mentioned I should only care about `en` translations. 

I've actually just tested what you mentioned and it should work just fine for other languages without `$1`. I just removed the `$1` from the English translation and it shows whatever text is there right now (see below on the screenshot). That means that other languages will see `Ethereum` until their strings contain `$1`, which should be done by our localization team.
<img width="351" alt="image" src="https://user-images.githubusercontent.com/80175477/114634656-e875af00-9c77-11eb-8b98-5d0315b4b5d9.png">

Reply:
That will flood sentry with errors. Our localization system expects each message to have the same number of substitutions across all locales

Reply:
Good to know. I've tried to replace all `Ethereum` occurrences with `$1`, but I don't see `Ethereum` in the ko/messages.json file: 
![image](https://user-images.githubusercontent.com/80175477/114635882-97b38580-9c7a-11eb-9cd8-448eb241679a.png)

Reply:
@dan437 Sorry for misleading you. When adding new translations, we only need to worry about `en` translations.When updating existing translations, if there is an obvious change to make to non-english translations, we can do that manually. If not, it is usually best to add a translation as new.

Reply:
According to google translate, 이더 리움 is korean for ethereum

Reply:
which appears to be the fifth set of characters from the left

Reply:
This does hint that localizing the actual network names is something we should do as well.

Reply:
That's interesting, thanks, Dan and Mark! I've replaced `이더리움` with `$1` and will make sure to localize network names as well.